src/Microsoft.DotNet.XHarness.Common/Execution/ProcessManager.cs

```diff
@@ -18,31 +19,22 @@ namespace Microsoft.DotNet.XHarness.Common.Execution
 {
     public abstract class ProcessManager : IProcessManager
     {
-
-        #region Abstract methods
+#region Abstract methods
 
         protected abstract int Kill(int pid, int sig);
         protected abstract List<int> GetChildrenPS(ILog log, int pid);
```

Comment:
not your change but this API is very poorly named.  If we're making massive changes, why not name it something like "GetChildProcessIds" or something that actually makes it clear what it's doing?  I don't know what a ChildrenPS is.

Reply:
Yeh, makes sense